src/call.c

```diff
@@ -83,7 +83,7 @@ struct call {
 	enum sdp_dir estvdir;      /**< Established video direction         */
 	bool use_video;
 	bool use_rtp;
-	char *user_data;           /**< User data related to the call       */
+	struct pl *user_data;      /**< User data related to the call       */
```

Comment:
`struct pl` is a very small struct so it could be better to just declare it as a compound
member and not a pointer:

```c
struct pl user_data
```